### ISSUE TITLE
[BUG] load_ogbn_mag() failed

### ISSUE THREAD

**Issue by wwt0805** [alibaba/GraphScope#1962]
I got a problem when using graphscope with stand-alone model and urgently need your help.

The code is as follows:
```
import graphscope
from graphscope.dataset import load_ogbn_mag
graphscope.set_option(show_log=False)  # enable logging
graph = load_ogbn_mag()
```
The error report is as follows:
```
Loading empty graph:   0%|          | 0/10 [00:00<?, ?it/s]
Loading empty graph: 100%|██████████| 10/10 [00:00<00:00, 599.66it/s]
Loading vertex labeled paper:   0%|          | 0/10 [00:00<?, ?it/s]
Loading vertex labeled paper:  20%|██        | 2/10 [00:00<00:00, 14.38it/s]
Loading vertex labeled paper:  60%|██████    | 6/10 [00:00<00:00, 25.22it/s]
Loading vertex labeled paper: 100%|██████████| 10/10 [00:00<00:00, 26.92it/s]
Loading vertex labeled paper: 100%|██████████| 10/10 [00:00<00:00, 25.30it/s]
Loading vertex labeled author:   0%|          | 0/10 [00:00<?, ?it/s]
Loading vertex labeled author: 100%|██████████| 10/10 [00:00<00:00, 96.44it/s]
Loading vertex labeled author: 100%|██████████| 10/10 [00:00<00:00, 96.18it/s]
Loading vertex labeled institution:   0%|          | 0/10 [00:00<?, ?it/s]
Loading vertex labeled institution: 100%|██████████| 10/10 [00:00<00:00, 129.13it/s]
Loading vertex labeled field_of_study:   0%|          | 0/10 [00:00<?, ?it/s]
Loading vertex labeled field_of_study: 100%|██████████| 10/10 [00:00<00:00, 119.92it/s]
*** Aborted at 1660211798 (unix time) try "date -d @1660211798" if you are using GNU date ***
PC: @                0x0 (unknown)
*** SIGBUS (@0x7faaa7abeff0) received by PID 297814 (TID 0x7fabf5ff3700) from PID 18446744072227647472; stack trace: ***
    @     0x7fac26f9b630 (unknown)
    @     0x7fac259c97ec __memcpy_ssse3_back
    @     0x7fac2773f298 vineyard::FixedSizeBinaryArrayBuilder::Build()
    @     0x7fac2774ac35 vineyard::FixedSizeBinaryArrayBaseBuilder::_Seal()
    @     0x7fac272306da vineyard::ObjectBuilder::Seal()
    @     0x7fac20334b1a _ZZN8vineyard13ArrowFragmentIlmE16AddNewEdgeLabelsERNS_6ClientEOSt6vectorISt10shared_ptrIN5arrow5TableEESaIS8_EERKS4_ISt3setISt4pairISsSsESt4lessISE_ESaISE_EESaISI_EEiENKUlS3_E2_clES3_
    @     0x7fac203cdfc4 _ZZN8vineyard11ThreadGroup7AddTaskIRZNS_13ArrowFragmentIlmE16AddNewEdgeLabelsERNS_6ClientEOSt6vectorISt10shared_ptrIN5arrow5TableEESaISA_EERKS6_ISt3setISt4pairISsSsESt4lessISG_ESaISG_EESaISK_EEiEUlS5_E2_JSt17reference_wrapperIS4_EEEEjOT_DpOT0_ENKUljSQ_DpOT_E_clIJS5_EEENS_6StatusEjSQ_S10_
    @     0x7fac203ce344 _ZNSt17_Function_handlerIFSt10unique_ptrINSt13__future_base12_Result_baseENS2_8_DeleterEEvENS1_12_Task_setterIS0_INS1_7_ResultIN8vineyard6StatusEEES3_EZNS1_11_Task_stateISt5_BindIFZNS8_11ThreadGroup7AddTaskIRZNS8_13ArrowFragmentIlmE16AddNewEdgeLabelsERNS8_6ClientEOSt6vectorISt10shared_ptrIN5arrow5TableEESaISO_EERKSK_ISt3setISt4pairISsSsESt4lessISU_ESaISU_EESaISY_EEiEUlSJ_E2_JSt17reference_wrapperISI_EEEEjOT_DpOT0_EUljS14_DpOT_E_jS13_S16_EESaIiEFS9_vEE6_M_runEvEUlvE_S9_EEE9_M_invokeERKSt9_Any_data
    @     0x7fac302a080b std::__future_base::_State_baseV2::_M_do_set()
    @     0x7fac26f9220b __pthread_once_slow
    @     0x7fac2026e7e2 _ZNSt13__future_base11_Task_stateISt5_BindIFZN8vineyard11ThreadGroup7AddTaskIRZNS2_13ArrowFragmentIlmE16AddNewEdgeLabelsERNS2_6ClientEOSt6vectorISt10shared_ptrIN5arrow5TableEESaISD_EERKS9_ISt3setISt4pairISsSsESt4lessISJ_ESaISJ_EESaISN_EEiEUlS8_E2_JSt17reference_wrapperIS7_EEEEjOT_DpOT0_EUljST_DpOT_E_jSS_SV_EESaIiEFNS2_6StatusEvEE6_M_runEv
    @     0x7fac27268290 execute_native_thread_routine
    @     0x7fac26f93ea5 start_thread
    @     0x7fac25971b0d __clone
    @                0x0 (unknown)
Loading edge labeled affiliated:   0%|          | 0/10 [00:00<?, ?it/s]2022-08-11 17:56:38,757 [ERROR][coordinator:433]: Engine RunStep failed, code: UNAVAILABLE, details: Connection reset by peer
2022-08-11 17:56:38,757 [ERROR][coordinator:433]: Engine RunStep failed, code: UNAVAILABLE, details: Connection reset by peer
Traceback (most recent call last):
  File "/root/test.py", line 12, in <module>
    graph = load_ogbn_mag()
  File "/usr/local/anaconda3/lib/python3.9/site-packages/graphscope/dataset/ogbn_mag.py", line 79, in load_ogbn_mag
    graph.add_vertices(os.path.join(prefix, "paper.csv"), "paper")
AttributeError: 'list' object has no attribute 'add_edges'

```
Please tell me how to solve it

**Comment by siyuan0322** [comment]
> __memcpy_ssse3_back
It seems you're running out of memory.
What environment are you using and how many memory do you have?
If you're in a docker environment, you need to set `shm-size` like `docker run --shm-size=4096m ....`

**Comment by sighingnow** [comment]
Closing as resolved as we already have warned about "shm-size too small" when running inside docker container.

**Comment by sighingnow** [comment]
> AttributeError: 'list' object has no attribute 'add_edges'

This incorrect error message has been addressed in #2094.
